src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final String DEFAULT_COLOR = Color.WHITE.name();
+    public static final double DEFAULT_RADIUS = 10;
+    public static final int MAX_RANDOM_NUMBER = 50;
+    public static final int MIN_RANDOM_NUMBER = 1;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int randomFigureIndex = random.nextInt(FIGURE_COUNT);
+        switch (randomFigureIndex) {
+            case 0:
+                return getRandomCircle();
+            case 1:
+                return getRandomIsoscelesTrapezoid();
+            case 2:
+                return getRandomRectangle();
+            case 3:
+                return getRandomRightTriangle();
+            case 4:
+            default:
+                return getRandomSquare();
+        }
+    }
+
+    private Circle getRandomCircle() {
+        return new Circle(colorSupplier.getRandomColor(), getDouble());
+    }
```

Comment:
```suggestion
    private Circle getRandomCircle() {
        return new Circle(colorSupplier.getRandomColor().toLowerCase(), getDouble());
    }
```
pass color already in lowecase
change in each method here

Reply:
But does it matter? Color changes to lowercase in Counstrucor of parent Figure Class.
![image](https://user-images.githubusercontent.com/112807544/189527614-e41bb2b6-6567-47f5-8e22-26b4ac5808f0.png)